apps/console/src/features/identity-providers/components/wizards/apple/apple-authentication-provider-create-wizard-content.tsx

```diff
@@ -63,6 +63,10 @@ interface AppleAuthenticationProviderCreateWizardContentPropsInterface extends I
      * @param values - Form values.
      */
     onSubmit: (values: AppleAuthenticationProviderCreateWizardFormValuesInterface) => void;
+    /**
+     * On focus callback.
+     */
+    setOnFocus: any;
```

Comment:
Shall we use the proper type instead of `any`?

Reply:
sure

Reply:
@pavinduLakshan all requested changes made